docs/questions/sharing/exporting-data.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Exporting data
```

Comment:
I feel like the the spreadsheet question exports and the pdf dashboard exports are qualitatively very different; in the first case, we're really exporting _data_, but in the the second case we're exporting a chart (even if the dashboard only has tables, the export won't contain _data_) so I'm not sure if the name fits. 

Maybe "Exporting results"? "Exporting data and results"?..